libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/us_ticker.c

```diff
@@ -68,66 +86,91 @@ void us_ticker_init(void)
     us_ticker_inited = 1;
 
     SystemCoreClockUpdate();
-    TimMasterHandle_CH3.PWM_CHn_PR = (GetSystemClock() / 1000000) -1;
-    TimMasterHandle_CH3.PWM_CHn_LR = 0xFFFFFFFF;
-    TimMasterHandle_CH3.PWM_CHn_PDMR = 1;
+    TimerInitType.PWM_CHn_PR = (GetSystemClock() / 1000000) -1;
+    TimerInitType.PWM_CHn_LR = 0xFFFFFFFF;
+    TimerInitType.PWM_CHn_PDMR = 1;
 
-    PWM_TimerModeInit(PWM_CH3, &TimMasterHandle_CH3);
-    PWM_CHn_Start(PWM_CH3);
+    PWM_TimerModeInit(TIMER_1, &TimerInitType);
+    PWM_CHn_Start(TIMER_1);
 }
 
 
 uint32_t us_ticker_read()
 {
     if (!us_ticker_inited) us_ticker_init();
-
-    return (PWM_CH3->TCR);
+    return (TIMER_1->TCR);
 }
 
 
 void us_ticker_set_interrupt(timestamp_t timestamp)
 {
     int32_t dev = 0;
+    
     if (!us_ticker_inited)
     {
         us_ticker_init();
     }
     
-    dev = (int32_t)(timestamp - (us_ticker_read() + 150));
-
+    dev = (int32_t)(timestamp - (us_ticker_read() + 160));
```

Comment:
2. If not add 160 at MBED_24
![aa](https://cloud.githubusercontent.com/assets/4919567/8712589/e4be87c0-2b98-11e5-910e-ba9668a006b4.png)
if add 80
![bb](https://cloud.githubusercontent.com/assets/4919567/8712605/1b57db7e-2b99-11e5-91ff-2e8a0fae583c.png)
if add 160
![cc](https://cloud.githubusercontent.com/assets/4919567/8712610/39fb0e84-2b99-11e5-873f-20c66b65315e.png)
And if add more value, other ticker tests are occurred timeout.
I assume it is because function jump delay.